src/components/AnnualReport/AGMDate.vue

```diff
@@ -1,68 +1,112 @@
 <template>
-  <v-card flat class="agm-date-container">
+  <v-card flat id="agm-date-container" v-if="isCoop">
     <div class="meta-container">
       <label>Annual General<br>Meeting Date</label>
 
       <v-form ref="form" class="value date">
+        <!-- date picker -->
         <v-menu
           ref="menu"
           v-model="menu"
           :close-on-content-click="false"
-          :nudge-right="40"
+          nudge-right="50"
           transition="scale-transition"
-          offset-y
           max-width="290"
         >
-          <template v-slot:activator="{ on }">
+          <template v-slot:activator="{ on, attrs }">
             <v-text-field
               data-test-id="agm-date-text"
+              class="pt-0"
               v-model="dateText"
               :disabled="noAgm"
               :rules="agmDateRules"
               label="Annual General Meeting Date"
               placeholder="Select your Annual General Meeting Date"
-              prepend-icon="mdi-calendar"
-              v-on="on"
+              append-icon="mdi-calendar"
               readonly
+              v-bind="attrs"
+              v-on="on"
               filled
             />
           </template>
           <v-date-picker
             data-test-id="agm-date-picker"
             v-model="datePicker"
-            :min=minDate
-            :max=maxDate
+            :min=arMinDate
+            :max=arMaxDate
             no-title
             @input="menu=false"
             @change="onDatePickerChanged($event)"
           />
         </v-menu>
 
-        <!-- only display this info if a date has been selected -->
-        <div class="validationErrorInfo" v-if="dateText">
+        <!-- restriction messages -->
+        <!-- date must have been selected first -->
+        <div v-if="dateText" class="restriction-messages mt-4">
           <span v-if="!allowCOA && !allowCOD">
-            You can not change your Registered Office Addresses or Directors in this Annual Report because your AGM
-            predates another filing that may have conflicting changes.
+            You cannot change your Registered Office Addresses or Directors in this Annual Report
```

Comment:
https://www.grammarly.com/blog/cannot-or-can-not/#:~:text=Here%20is%20a%20quick%20summary,.%20.%20.%20but%20also.%E2%80%9D